torch/optim/lr_scheduler.py

```diff
@@ -1,23 +1,42 @@
 import types
 import math
-from torch._six import inf
-from functools import partial, wraps
 import warnings
+from collections import Counter
+from functools import partial, wraps
 from bisect import bisect_right
+from torch._six import inf
 
 from .optimizer import Optimizer
 
 
+EPOCH_DEPRECATION_WARNING = (
+    "The behavior of schedulers has been changed from their closed form to their "
```

Comment:
this statement is not comprehensible (I have no idea what it means after reading it)

Reply:
Thanks for the feedback. Please let me know if the new version is clearer.

Reply:
Thanks for the feedback. Please let me know if the new version is clearer.